apps/src/p5lab/AnimationPicker/AnimationUploadButton.jsx

```diff
@@ -196,16 +104,22 @@ export function UnconnectedAnimationUploadButton({
   }
 
   function cancelUpload() {
-    setRestrictedShareConfirmed(false);
-    setNoPIIConfirmed(false);
     setIsUploadModalOpen(false);
     exitedUploadWarning();
   }
 
   return (
     <>
-      {renderUploadModal()}
-      {renderPublishedWarningModal()}
+      <ImageUploadModal
```

Comment:
When I first read the name of this file, I expected that it woud only render the button. A bit out of scope of this PR, but what do you think of renaming this component, maybe to `AnimationUploader`?

Reply:
Totally agree -- I have a couple follow-ups here, I'll try and sneak in a rename in one of those.